.contributing/the-process.md

```diff
@@ -158,4 +154,24 @@ Notes on the various tools 
 
 -   Google docs are great for creating longer, more complex documents (eg. with diagrams and a TOC)
 
--   Asana Discussions are great for capturing Q&A type conversations (a bit like StackOverflow)
\ No newline at end of file
+-   Asana Discussions are great for capturing Q&A type conversations (a bit like StackOverflow)
+
+
+Key
+====
+
+- "IS" : input session
+- "user flow folder": folder created in user flow step 1, titled: "x. *Use Case* - User Flow files - *Section* (CL)"
+- "input task": 
+  - Create a task in Asana with title "IS: Lofi Sketch < x. Use Case" and:
```

Comment:
Can we make this "IS: _title of Google doc_" instead

Reply:
Just to clarify - I would be happy with just adding an "eg.", and also happy with @T-Davis suggestion :-)

Reply:
yess!!